project2/index.html

```diff
@@ -0,0 +1,98 @@
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles.css">
+    <link href="https://fonts.googleapis.com/css?family=Merriweather:400,700&display=swap" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css?family=Noto+Sans+JP:500,700&display=swap" rel="stylesheet">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Document</title>
+</head>
+
+<body>
+
+    <section class="wholeting">
+        <header>
+            <h2 class="title"> Shrimp & Grits </h2>
+            <h4 class="subhead"> 30 Minute Meal </h4>
+        </header>
+        <section class="second">
+            <h3 class="secondheader"> Your Ingredients </h3>
+            <div class="ingredients">
+                <ul>
+                    <li>
+                        3/4th Teaspoon Salt
+                    </li>
+                    <li>
+                        1/2 Teaspoon Pepper
+                    </li>
+                    <li>
+                        2% milk
+                    </li>
+                    <li>
+                        1/3 cup butter, cubed
+                    </li>
+                    <li>
+                        1/2 teaspoon pepper
+                    </li>
+                    <li>
+                        3/4 cup old-fashioned grits
+                    </li>
+                    <li>
+                        1 cup shredded cheese
+                    </li>
+                    <li>
+                        4 chopped onions
+                    </li>
+                    <li>
+                        1 teaspoon blackened seasoning
+                    </li>
+                    <li>
+                        3 garlic cloves
+                    </li>
+                    <li>
+                        1lb medium shrimp </li>
+                    <li>
+                        8 chopped bacon strip
+                    </li>
+                </ul>
+            </div>
+        </section>
+
+        <section class="third">
+            <h3 class="thirdheader"> Quick Directions </h3>
+
+            <p class="copy">
+                In a large saucepan, bring the broth, milk, butter, salt and pepper to a boil. Slowly stir in grits.
+                Reduce
+                heat. Cover and cook for 12-14 minutes or until thickened, stirring occasionally. Stir in cheese until
+                melted.Set aside and keep warm.In a large skillet, cook bacon over medium heat until crisp. Remove to
+                paper
+                towels with a slotted spoon;
+                drain,reserving 4 teaspoons drippings. Saute the shrimp, garlic and seasoning in drippings until shrimp
+                turn
+                pink.
+                Serve with grits and sprinkle with onions.
+            </p>
+        </section>
+
+        <section class="bottom">
+            <ul class="social">
```

Comment:
Think a bit more about the links you're using here -- if one page had three recipes on it, would it make sense to have the "contact us" and "newsletter" links repeated three times?